euclidean_gcd/euclidean_gcd.js

```diff
@@ -0,0 +1,25 @@
+function euclidean_gcd_recursive(first, second) {
```

Comment:
Please explain the standard naming convention, I've just followed what I got from the C file of this algo ? That will mean C file will require change too ?